File: Source/ses_io/src/wrappers/fortran/wrap_ses_write_setup.c

#include "ses_defines.h"
#include "ses_globals.h"
#include "ses_internals.h"
#include "ses_externs.h"

#include "stdio.h"
#include <stdlib.h>

#include <stdio.h>

#undef DEBUG_WRAP

#ifdef LC_UNDER
ses_error_flag ses_write_setup_(ses_file_handle* pt_handle, ses_material_id* pt_mid, ses_table_id* pt_tid,  ses_number nr, ses_number nt, ses_number ntab) {
#else

#ifdef UC_UNDER
ses_error_flag SES_WRITE_SETUP_(ses_file_handle* pt_handle, ses_material_id* pt_mid, ses_table_id* pt_tid,  ses_number nr, ses_number nt, ses_number ntab) {
#else

#ifdef LC_NOUNDER
ses_error_flag ses_WRITE)setup(ses_file_handle* pt_handle, ses_material_id* pt_mid, ses_table_id* pt_tid,  ses_number nr, ses_number nt, ses_number ntab) {
#else

#ifdef UC_NOUNDER
ses_error_flag SES_WRITE_SETUP(ses_file_handle* pt_handle, ses_material_id* pt_mid, ses_table_id* pt_tid,  ses_number nr, ses_number nt, ses_number ntab) {
#endif

#endif
#endif
#endif

  ses_error_flag return_value = SES_NO_ERROR;

  ses_file_handle the_handle = *pt_handle;
  ses_material_id mid = *pt_mid;
  ses_table_id tid = *pt_tid;

  return_value = ses_write_setup(the_handle, mid, tid, nr, nt, ntab);
  return return_value;

}

 

File: Source/ses_io/src/wrappers/fortran/wrap_ses_write_setup.f90

#undef DEBUG_WRAP

SES_ERROR_FLAG function ses_write_setup_f90(the_handle, the_mid, the_tid, nr, nt, ntab)

   implicit none

   SES_FILE_HANDLE, INTENT(IN) :: the_handle
   SES_MATERIAL_ID, intent(in) :: the_mid
   SES_TABLE_ID, intent(in) :: the_tid
   SES_NUMBER, intent(in) :: nr
   SES_NUMBER, intent(in) :: nt
   SES_NUMBER, intent(in) :: ntab

   SES_ERROR_FLAG :: return_value, ses_write_setup


   return_value = ses_write_setup(the_handle, the_mid, the_tid, nr, nt, ntab)

   ses_write_setup_f90 = return_value

 
end function ses_write_setup_f90


